### DISCUSSION TITLE
Feedback - Dcm2bids 3.0.0

### DISCUSSION THREAD

**Discussion by arnaudbore** [UNFmontreal/Dcm2Bids#240] [General]
We are welcoming any comments on the new release ! Go

**Comment by seaguldee** [comment]
Hi, we are running into a problem in that we've set up a detailed workflow with dcm2bids v 2. but need it to run on the most current version of dcm2niix (April 2023) for some unexpected issues in some additional Phillips scans.  Wondering if it's possible to get a v 2 container with a more updated dcm2niix added to docker?  We'd like to avoid modifying our config files for version 3.0 as there are a ton of MRI scans.  I see that there is a [2.1.9_latest_dcm2niix] version on docker but this has an older dcm2niix in it.  Any chance someone could be really nice to us and upload v2 dcm2bids with updated dcm2niix (from later than April 2023)?

**Reply by arnaudbore** [reply]
I just pushed 2.1.9_dcm2niix_v1.0.20230411 - Tell me if it fixes your issue.

**Reply by arnaudbore** [reply]
@seaguldee :arrow_up:

**Comment by RenchZhao** [comment]
Good day!
Sometimes my dataset has many subjects to perform dcm2niix. They become subdirectories of my dataset directory.
I changed the code on 

line 101 of dcm2niix_gen.py to 
self.sidecarFiles = glob(os.path.join(self.output_dir, "**","*.json"),recursive=True)

so that all subdirectories files generated by dcm2niix can be found.
